mne_bids/path.py

```diff
@@ -94,19 +107,18 @@ def entities(self):
             ('acquisition', self.acquisition),
             ('run', self.run),
             ('processing', self.processing),
-            ('recording', self.recording),
             ('space', self.space),
-            ('prefix', self.prefix),
-            ('suffix', self.suffix)
+            ('recording', self.recording),
+            ('kind', self.kind),
         ])
 
-    def __str__(self):
-        """Return the string representation of the path."""
+    @property
+    def basename(self):
+        """Return the basename of the BIDS Path."""
```

Comment:
Drop the "return", as it's not an ordinary function anymore. Also I think we can drop the "BIDS Path" part, it seems redundant?
```suggestion
        """The basename."""
```

or

```suggestion
        """The path basename."""
```